domain/infosync/info.go

```diff
@@ -54,17 +55,24 @@ const (
 	InfoSessionTTL = 10 * 60
 	// ReportInterval is interval of infoSyncerKeeper reporting min startTS.
 	ReportInterval = 30 * time.Second
+	// TopologyInformationPath means etcd path for storing topology info.
+	TopologyInformationPath = "/topology/tidb/"
+	// TopologyTimeToLive is ttl for topology.
+	TopologyTimeToLive = 60 * time.Second
```

Comment:
1 minute is too long. How about 30 seconds TTL + 15 seconds refresh

Reply:
I think both are OK...

Reply:
I change it to 45 secs now.